app/schemas/users.js

```diff
@@ -71,17 +71,11 @@ exports.getUser = {
   page: {
     in: ['query'],
     isInt: true,
-    isEmpty: {
-      negated: true,
-      errorMessage: ErrorMessages.PAGE_REQUIRED
-    }
```

Comment:
You shouldn't remove the message for this validation. You can do the following:
`isInt: { errorMessage: ErrorMessages.PAGE_REQUIRED}`

Reply:
Remember that the error message would already be different.